### ISSUE TITLE
Liquid concrete piping 

### ISSUE THREAD

**Issue by areyesca** [Deadlock989/IndustrialRevolution#156]
Hello,

When i´m preparing the hardware to perform the concrete recipe (liquid concrete + iron rods), you need to previously prepare the liquid concrete in a mixer, getting a liquid output. Then you need to transfer liquid concrete to an assembler to prepare the concrete. The only way to do it is this one, directly linking the mixer and the assembler:

![imagen](https://user-images.githubusercontent.com/6408323/211092843-7df669f4-2966-49e7-9156-6a743c218ee5.png)

I cannot use the water pipe to transfer liquid concrete as is water the unique liquid that you can place in a copper pipe. But, i can use iron pipes. 

Can you modifiy the allowed liquid to include liquid concrete as valid fluid in copper pipes? Just for aestetic and keep all shiny copper-like!

Best regards!

**Comment by Deadlock989** [comment]
Unfortunately pipes and pipe connections can only have one filter (cf. #148).

However this does need some sort of attention as it wasn't really intended. Liquid concrete was added quite late in developement and I didn't spot the consequence of having it unlocked before electric assemblers. Will have to think about it.

**Comment by areyesca** [comment]
Thank you so much for your fast answer... rathen than an issue this is a suggestion.

Keep doing this great job!

**Comment by Deadlock989** [comment]
There are a few solutions to this and I don't particularly like any of them.

- **Status quo** - do nothing. It is perfectly possible to make concrete in steam machines using the direct connection method shown in the pic above, or just use iron pipes if you don't care about the aesthetic.
- **Remove water filter from copper pipes** - this would make them capable of transporting any fluid, including steam and molten metals. They are more expensive than steam pipes but it would still make steam pipes verge on being obsolete. Copper pipes have less health and resistances than iron pipes but still, there is no need for two all-purpose pipes (this is why IR2 steel pipes were removed). Another way to distinguish them would be to fiddle with their volume which has a very indirect impact on their throughput but the fluid system is pretty arcane and I'm not confident this would work or even be interesting.
- **Add an all-purpose bronze pipe** - this would be easy to do graphics-wise and they would sit better with mixer-to-assembler connections but it's menu clutter, no space for them; also again no actual need for a second tier of all-purpose pipe.
- **Make concrete require electric assemblers** - this would break some saved games. It has a knock-on research cost effect on every tech that follows concrete because it is currently used to make oil refineries and chemical plants. Crude oil processing tech cost jumps from 175 to 250 (two levels), steel tech cost from 200 to 250 (one level), everything after steel goes up one level, several of the oil processing techs and related techs go up two levels until that branch converges with steel again.
- **Remove liquid concrete** - go back to the vanilla system of having refined concrete made from regular concrete. The main reason I did this in the first place was to avoid having both iron and steel rods end up in the same item.
